### DISCUSSION TITLE
NAM expanded area

### DISCUSSION THREAD

**Discussion by afernandezody** [NOAA-GSL/GSL-Regional-Model-Forum#45] [General]
Hello,
I'm looking at https://www.emc.ncep.noaa.gov/mmb/namgrids/g212.12km.jpg and was wondering which is the expanded NAM area. I've always used the standard NAM area (the one with the bold borderline) so I'm not even sure if there are available data for the expanded area. I was wondering if someone could clarify this.
Thanks

**Comment by GeoffManikin-NOAA** [comment]
The dashed line is the full domain of the 12 parent NAM.     The bold
line indicates the domain of various resolutions (20, 40, and ~80 km) of a
subsetted regional grid for those who don't want to download the full
domain.

    Data is available for the full domain at either 12 or 32 km.

    There is a listing of output grids and links to data on the NCO NAM
products inventory page. <https://www.nco.ncep.noaa.gov/pmb/products/nam/>
   The full domain grids are identified as #218 (12 km) and #221 (32 km).
The regional gridsl are #215 (20 km), #212 (40 km), and #211 (81 km).


----------------------------------------------------
*Geoff Manikin*
Chief, Statistical Modeling Division
Meteorological Development Laboratory
Office of Science and Technology Integration
National Weather Service
C: 301-801-1421

**Comment by afernandezody** [comment]
Thanks for the feedback. I'm only interested in the 12-km resolution data and downloading from the S3 bucket. The thing is that the notation is slightly different (if my recollection is correct). I've been using the files denoted with 'awphys', which cover the domain inside the bold line. If I want to use the full domain at 12-km resolution, I'm having a hard time identifying which files hold these data.

**Comment by GeoffManikin-NOAA** [comment]
Correct - the 'awphys' fies are 12 km but only cover that inside
domain.

    I need to correct what I wrote earlier:    There is a listing of output
grids and links to data on the NCO NAM products inventory page.
<https://www.nco.ncep.noaa.gov/pmb/products/nam/>     The primary full
domain grid is #221 (32 km).   The regional grids are #218 (12 km),  #215
(20 km), #212 (40 km), and #211 (81 km).  (I previously identified 218 as a
full domain grid.  It is not.)

    The files labeled 'awip32' are full domain but are 32 km.    I believe
that the 'bgrdsf' flies are 12 km files covering the full domain, but they
contain mostly surface fields and are on a rotated lat/lon grid.   Someone
can correct me if a full domain, 12 km, non-rotated lat/lon grid is
available.


----------------------------------------------------
*Geoff Manikin*
Chief, Statistical Modeling Division
Meteorological Development Laboratory
Office of Science and Technology Integration
National Weather Service
C: 301-801-1421

**Reply by afernandezody** [reply]
That pretty much agrees with what I noticed yesterday night while browsing the bucket. It's a bit odd (for someone who has not been involved in the project) that the data for the full domain with the fine resolution is not available, but I guess that there is probably some reason.
